### ISSUE TITLE
install istio on vcluster stuck  in pending

### ISSUE THREAD

**Issue by ju187** [loft-sh/vcluster#141]
Following the quick start to create a vcluster in a eks cluster.  When install istio into this vcluster, the istio pod stuck at pending state.  logs show no detailed status of the pod

**Comment by FabianKramm** [comment]
@ju187 thanks for opening this issue! Is there a pod in the underlying cluster created? Is the pod in the underlying cluster pending as well? Any events shown through `kubectl describe`?

**Comment by ju187** [comment]
Yes, the pod was created but there is no event when I did a describe.
Tony

    On Thursday, September 30, 2021, 12:26:47 PM PDT, Fabian Kramm ***@***.***> wrote:  
 
 


@ju187 thanks for opening this issue! Is there a pod in the underlying cluster created? Is the pod in the underlying cluster pending as well? Any events shown through kubectl describe?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
